api/v1alpha1/volumereplicationgroup_types.go

```diff
@@ -152,17 +159,46 @@ const (
 
 const KubeObjectProtectionCaptureIntervalDefault time.Duration = 5 * time.Minute
 
+type ResourceCaptureInstance struct {
```

Comment:
I suggest `Instance` be renamed `Spec` and `Group` be added: `ResourceCaptureGroupSpec`